arches_for_science/migrations/0001_initial.py

```diff
@@ -8,6 +8,7 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
+        ("models", "10121_workflowhistory"),
```

Comment:
Open to other ways of doing this, but without this I was finding:
```py
django.db.utils.IntegrityError: null value in column "name" of relation "django_content_type" violates not-null constraint
DETAIL:  Failing row contains (1, null, models, bulkindexqueue).
```

Reply:
When testing, I didn't see this error after loading from scratch.  I went back to zero, removed the dependencies, and ran them forward and the migration applied without issue.

Reply:
Issue is that core is referenced in the second migration.  There's a load order issue that can occur when loading from scratch.  This solution seems good to me.